airflow/contrib/operators/dataproc_operator.py

```diff
@@ -523,7 +527,10 @@ def execute(self, context):
         job = hook.create_job_template(self.task_id, self.dataproc_cluster, "hiveJob",
                                        self.dataproc_properties)
 
-        job.add_query(self.query)
+        if self.query is None:
```

Comment:
Just wanted to be consistent with the implementation in the Pig job operator (line 451).
But if you feel it's better, I can change it like that in all the three occurrences (line 451, 530, 609).
Thoughts?